java/daikon/FileIO.java

```diff
@@ -1787,6 +1791,12 @@ public static void process_sample(
         throw new RuntimeException(
             "Bad program point name " + ppt.name + " is a combined exit point name");
       }
+
+      if (ppt.ppt_name.isExceptionPoint() && ppt.ppt_name.isCombinedThrowPoint()) {
```

Comment:
The lack of parallelism between the names "isExceptionPoint()" and "isCombinedThrowPoint()" is confusing.  Please use the same term, either "Exception" or "Throw", in both.